near-sdk/src/collections/set.rs

```diff
@@ -63,27 +63,21 @@ impl<T> Set<T> {
         env::storage_has_key(&index_lookup)
     }
 
-    /// Adds a value to the set. If the set did not have this value present, `true` is returned. If
-    /// the set did have this value present, `false` is returned. Note, the elements that have the same hash value are undistinguished by
-    /// the implementation.
+    /// Adds a value to the set.
+    /// If the set did not have this value present, `true` is returned.
+    /// If the set did have this value present, `false` is returned.
+    /// Note, the elements that have the same hash value are undistinguished by the implementation.
```

Comment:
Since we do not hash state keys this comment is incorrect.

Reply:
removed